docs/ldml/tr35.md

```diff
@@ -1350,38 +1350,39 @@ Hybrid locales have intermixed content from 2 (or more) languages, often with on
 This tends to work better in implementations that don't yet handle the -t- extension.
 
 Examples:
 
-<!-- HTML: no header, colspan -->
-<table><tbody>
-<tr><td>hi-t-<u>en-h0-hybrid</u></td>   <td>Hinglish</td>   <td>Hindi-English hybrid locale</td></tr>
-<tr><td>ta-t-<u>en-h0-hybrid</u></td>   <td>Tanglish</td>   <td>Tamil-English hybrid locale</td></tr>
-<tr><td>ba-t-<u>en-h0-hybrid</u></td>   <td>Banglish</td>   <td>Bangla-English hybrid locale</td></tr>
-<tr><td colspan="3">…</td></tr>
-<tr><td>en-t-<u>hi-h0-hybrid</u></td>   <td>Hinglish</td>   <td>English-Hindi hybrid locale</td></tr>
-<tr><td>en-t-<u>zh-h0-hybrid</u></td>   <td>Chinglish</td>  <td>English-Chinese hybrid locale</td></tr>
-<tr><td colspan="3">…</td></tr>
-</tbody></table>
+|Locale ID			| Base script	| Hybrid name	| Description									|
+|-------------------------------|---------------|---------------|-------------------------------------------------------------------------------|
+|hi-t-***en-h0-hybrid***	| Deva		| Hinglish	| Hindi-English hybrid where the script is Devanagari*				|
+|hi-Latn-t-***en-h0-hybrid***	| Latin		| Hinglish	| Hindi-English hybrid where the script is Latin*				|
+|hi-Latn			| Latin		| Hinglish	| Hindi written in Latin script in practice usually a hybrid with English	|
+|ta-t-***en-h0-hybrid***  	| Tamil		| Tanglish	| Tamil-English hybrid where the script is Tamil*				|
+|...																		||
+|en-t-***hi-h0-hybrid***	| Latin		| Hinglish	| English-Hindi hybrid	where the script is Latin*				|
+|en-t-***zh-h0-hybrid***	| Latin		| Chinglish	| English-Chinese hybrid where the script is Latin*				|
+|...																		||
+
+* When used as a request for international services (such as date formatting), the request is for everything to be in the base script if possible. When used to tag arbitrary content on a coarse level, the expectation is that it be the predominant script — that is, there may be certain passages or phrases that are in the other script but are not tagged on a fine-grained level.
 
-> _Note: The [unicode_language_id](#unicode_language_id) should be the language used as the ‘scaffold’: for the fallback locale for internationalization services, typically used for more of the core vocabulary/structure in the content. Thus Hinglish should be represented as hi-t-h0-en where Hindi is the scaffold, and as en-t-h0-hi where English is._
+> _Note: The [unicode_language_id](#unicode_language_id) should be the language used as the ‘scaffold’: for the fallback locale for internationalization services, typically used for more of the core vocabulary/structure in the content. Thus where Hindi is the scaffold, Hinglish should be represented as hi-t-en-h0 (when written in Devanagari) or hi-Latn-en-h0 (when written in Latin characters). Where English is the scaffold, Hinglish should be represented as en-t-hi-h0 (or possibly en-Deva-t-hi-h0)._
```

Comment:
The syntax here went back to ill-formed language tags.
```suggestion
> _Note: The [unicode_language_id](#unicode_language_id) should be the language used as the ‘scaffold’: for the fallback locale for internationalization services, typically used for more of the core vocabulary/structure in the content. Thus where Hindi is the scaffold, Hinglish should be represented as hi-t-en-h0-hybrid (when written in Devanagari script) or hi-Latn-t-en-h0-hybrid (when written in Latin characters). Where English is the scaffold, Hinglish should be represented as en-t-hi-h0-hybrid (or possibly en-Deva-t-hi-h0-hybrid)._
```

Reply:
Good catches, thanks